templates/Bluecadet App/src/_TBOX_PREFIX_App.cpp

```diff
@@ -0,0 +1,70 @@
+// Cinder
+#include "cinder/app/App.h"
+#include "cinder/app/RendererGl.h"
+#include "cinder/gl/gl.h"
+
+// Bluecadet
+#include "bluecadet/core/BaseApp.h"
+#include "bluecadet/views/TouchView.h"
+
+using namespace std;
+using namespace ci;
+using namespace ci::app;
+using namespace bluecadet::core;
+using namespace bluecadet::views;
+using namespace bluecadet::touch;
+
+class _TBOX_PREFIX_App : public BaseApp {
+public:
+	static void prepareSettings(ci::app::App::Settings *settings);
+	void setup() override;
+	void update() override;
+	void draw() override;
+};
+
+void _TBOX_PREFIX_App::prepareSettings(ci::app::App::Settings *settings) {
+	// Optional: Override the shared settings manager instance with your subclass
+	// SettingsManager::setInstance(myApp::MyAppSettingsManager::getInstance());
+
+	// Initialize the settings manager with the cinder app settings and the settings json
+	SettingsManager::getInstance()->setup(
+		settings,
+		ci::app::getAssetPath("settings.json"),
+		[](SettingsManager *manager) {
+			// Optional: Override json defaults at runtime
+			manager->mFullscreen = false;
+			manager->mWindowSize = ivec2(1280, 720);
+		});
+}
+
+void _TBOX_PREFIX_App::setup() {
+	BaseApp::setup();
+	BaseApp::addTouchSimulatorParams();
```

Comment:
I believe @adielfernandez made this into a JSON parameter. @adielfernandez Did you ever commit this to the views block?

Reply:
Yup! The change is in this commit: https://github.com/bluecadet/Cinder-BluecadetViews/commit/2614a01d9589b58209fefba957fbdb3229261494

Reply:
Great! @shi-weili Can you take this line out then and make sure the JSON includes that change. 

@adielfernandez could you do me a favor and grab the SettingsManager + json additions I made to Wetland Data to set the window position and fold it into the views block?

Reply:
Done.

Reply:
@benjaminbojko Sure thing! Will pull the changes from WD and fold them into Views when I get back to NYC.